GAE/src/org/waterforpeople/mapping/app/web/EnvServlet.java

```diff
@@ -143,7 +143,7 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
         if (props.get(CADDISFLY_TESTS_FILE_URL_KEY) == null
                 || props.get(CADDISFLY_TESTS_FILE_URL_KEY).isEmpty()) {
             props.put("caddisflyTestsFileUrl",
-                    "https://akvoflow-public.s3.amazonaws.com/caddisfly-tests.json");
+                    "https://s3-eu-west-1.amazonaws.com/akvoflow-public/caddisfly-tests-v2.json");
```

Comment:
how come the change to a different bucket name?

Reply:
The previous file is also used in lumen so with the new properties in the json we opted to upload and use a new file to avoid breaking anything

Reply:
ahh my mistake. I see its just the alias of the bucket... ok 👍